extensions/gopass/src/gopass.ts

```diff
@@ -39,6 +43,29 @@ async function list({ limit = -1, prefix = "", directoriesFirst = false, stripPr
     .then((data) => (directoriesFirst ? sortDirectoriesFirst(data) : data));
 }
 
+async function pwgen(
+  type: string,
+  length: number,
+  symbols: boolean,
+  digits: boolean,
+  capitalize: boolean,
+  numbers: boolean
+): Promise<string[]> {
+  if (type === "xkcd") {
+    return await gopass([
```

Comment:
The `await` here and in line 64 is unnecessary. This is a very minor nitpick 😅

Reply:
done